### PR TITLE
Fixing collect_tex.py for new ravenframework changes.

### LINKED ISSUE #51: [DEFECT] Documentation no longer builds on macOS
--------
Defect Description
--------

**Describe the defect**

##### What did you expect to see happen?

I imagine due to the recent change in module structure in RAVEN and related plugins, we are now seeing the documentation fail to build due to relative import errors. 

Here is the error I get when trying to run `make` on macOS:

```
chmod u+x script/copy_tex.sh
script/copy_tex.sh
python script/generate_user_manual.py
Traceback (most recent call last):
  File "/Users/mcdodj/Documents/projects/TEAL/doc/script/generate_user_manual.py", line 9, in <module>
    import collect_tex
  File "/Users/mcdodj/Documents/projects/TEAL/doc/script/collect_tex.py", line 15, in <module>
    import CashFlows
  File "/Users/mcdodj/Documents/projects/TEAL/src/CashFlows.py", line 32, in <module>
    from ..src import Amortization
ImportError: attempted relative import with no known parent package
make: *** [populate] Error 1
```

**Describe how to Reproduce**
1.  Run `make` on macOS
2. 
3. 
4. 

**Screenshots and Input Files**
Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

**Platform (please complete the following information):**
 - Version: [e.g. 22]
 - Dependencies Installation: [CONDA or PIP]

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [ ] 1. Is it tagged with a type: defect or task?
- [ ] 2. Is it tagged with a priority: critical, normal or minor?
- [ ] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [ ] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [ ] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by joshua-cogliati-inl**
--------
Pull Request Description
--------
##### What issue does this change request address?
Closes #51 

##### What are the significant changes in functionality due to this change request?
Updates how to set TEAL and ravenframework paths.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/HERON/wiki/Code-Standards) for details.
- [ ] 4. Automated Tests should pass.
- [x] 5. If significant functionality is added, there must be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large tes.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [ ] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added, the the analytic documentation must be updated/added.
- [x] 9. If any test used as a basis for documentation examples have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

File: doc/script/collect_tex.py [hunk 1]
```diff
@@ -7,16 +7,23 @@
...
 sys.path.append(teal_path)
-import CashFlows
-# import _utils as hutils
-# framework_path = hutils.get_raven_loc()
-# sys.path.append(framework_path)
-# from utils import InputData
+try:
+  from ravenframework.util import InputData
+except ModuleNotFoundError:
+  framework_path = os.path.join(teal_path, '..', 'ravenframework')
```
Comment (PaulTalbot-INL) [inline doc/script/collect_tex.py]: I think TEAL still outlaws underscore variable names, unlike HERON.

**Review by PaulTalbot-INL — CHANGES_REQUESTED** [review]
Changes are in line with similar changes in TEAL and other plugins. Just one naming convention violation.

File: doc/script/collect_tex.py (hunk 1 above)
Comment (joshua-cogliati-inl) [reply doc/script/collect_tex.py]: Fixed.

**Review by PaulTalbot-INL — DISMISSED** [review]
new variables introduced follow conventions.

**Comment by PaulTalbot-INL** [comment]
Note this should not automatically close #51, until we can show documentation builds on the three main OS, since we do not currently check documentation builds in the regression tests.

**Review by PaulTalbot-INL — DISMISSED** [review]

**Comment by joshua-cogliati-inl** [comment]
I checked that it works in Linux.

**Comment by PaulTalbot-INL** [comment]
It seems to have compiled fine on mac; no errors.

**Comment by dgarrett622** [comment]
I checked that it works for Windows.

**Review by PaulTalbot-INL — APPROVED** [review]

**Comment by PaulTalbot-INL** [comment]
ok. merging and closing the associated issue.

**Pull request merged**